plugins/modules/site_workflow_manager.py

```diff
@@ -1287,15 +1222,20 @@ def validate_site_input_data(self, config):
                     if not (isinstance(longitude, (float, int)) and -180 <= longitude <= 180):
                         errormsg.append("Invalid longitude. Valid range is -180 to +180.")
 
-                if not (latitude and longitude or address):
-                    errormsg.append("Either latitude/longitude or address is required.")
-                    self.log("Missing required latitude/longitude or address for building.", "ERROR")
-                elif (latitude and not longitude) or (not latitude and longitude):
-                    errormsg.append("Either Latitude or longitude is missing in the given playbook")
+                if self.compare_dnac_versions(self.get_ccc_version(), "2.3.7.6") >= 0:
+                    if not (latitude and longitude or address):
+                        errormsg.append("Either latitude/longitude or address is required.")
+                        self.log("Missing required latitude/longitude or address for building.", "ERROR")
+                    elif (latitude and not longitude) or (not latitude and longitude):
+                        errormsg.append("Either Latitude or longitude is missing in the given playbook")
+                else:
+                    if not (latitude and longitude):
+                        errormsg.append("latitude and longitude is required.")
```

Comment:
errormsg.append("Latitude and longitude are required.")

Reply:
Modified